src/app/utils/ProxifyUrl.js

```diff
@@ -8,9 +8,14 @@
  * <proxy>/{int}x{int}/[<proxy>/{int}x{int}/]<proxy>/<file url>
  * @type {RegExp}
  */
-const rProxyDomain = /^http(s)?:\/\/steemit(dev|stage)?images.com\//g;
-const rProxyDomainsDimensions = /http(s)?:\/\/steemit(dev|stage)?images.com\/([0-9]+x[0-9]+)\//g;
-const NATURAL_SIZE = '0x0/';
+const rProxyDomain = /^http(s)?:\/\/steemit(dev|stage)?images.com\//g
```

Comment:
Let’s make a URL parsing class that takes a string and has methods like isImageProxyUrl() and isValidURL() (from a better url validation library that isn’t maintained by us) and imageProxyDimensions() and suchlike, so that all URL-parsing regex are in a single place.

Reply:
That sounds good-ish, but solving it all the way is beyond the bounds of this issue.

The only functional addition to ProxifyUrl is the imageProxy. It was added to reduce direct dependence on window.$STM_Config (a terrible thing) and to begin the process of creating that "URL parsing class".

What we have here is a step in the right direction; further changes are not warranted for this merge.